example/reactotron.ts

```diff
@@ -13,3 +13,17 @@ Reactotron.useReactNative({
 });
 
 Reactotron.connect();
+
+declare global {
+  interface Console {
+    tron: typeof Reactotron;
+  }
+}
+
+console.tron = Reactotron;
+const consoleLog = console.log;
+
+console.log = (message: string, ...optionalParams: any[]) => {
+  consoleLog(message, ...optionalParams);
+  console.tron.log?.(message, ...optionalParams);
+};
```

Comment:
This is safe because this whole file is lazy loaded with the following code in `AppDemo.txt`:

```TSX
if (__DEV__) {
  import('./reactotron').then(() => console.log('Reactotron Configured'));
}
```